osmaxx-py/osmaxx/excerptexport/models/output_file.py

```diff
@@ -28,7 +28,7 @@ def download_file_name(self):
             'id': str(self.public_identifier),
             'name': os.path.basename(self.file.name) if self.file else None,
             'date': self.create_date.strftime("%F"),
-            'excerpt_name': self.extraction_order.excerpt.name.replace(" ", ""),
+            'excerpt_name': self.extraction_order.excerpt_name,
```

Comment:
I think it's better to keep it. Nobody knows what we will do later. Otherwise we may struggle with spaces in names later.